### PR TITLE
Update climate_HassClimateGetTemperature.yaml

### PR DESCRIPTION
**Pull request by Kalma-House**
More ways ta ask about temperature.

### CODE REVIEW

File: tests/es/climate_HassClimateGetTemperature.yaml [hunk 1]
```diff
@@ -3,11 +3,20 @@ tests:
...
     intent:
       name: "HassClimateGetTemperature"
   - sentences:
       - "¿cuál es la temperatura en la cocina?"
       - "¿qué temperatura hace en la cocina?"
+      - "¿qué temperatura hay en la cocina?"
+      - "¿cuántos grados hace en la cocina?"
+      - "¿cuántos grados hay en la cocina?"
+      - "¿a cuántos grados estamos en la cocina?"
+      - "¿a cuántos grados están en la cocina?"
```
Comment (cibernox) [inline tests/es/climate_HassClimateGetTemperature.yaml:19]: - "¿A qué temperatura está la cocina?"

Comment (Kalma-House) [reply tests/es/climate_HassClimateGetTemperature.yaml:20]: Added. Thank you.

Comment (cibernox) [reply tests/es/climate_HassClimateGetTemperature.yaml:20]: Missing accent in `qué`

Comment (cibernox) [reply tests/es/climate_HassClimateGetTemperature.yaml:20]: I told you I am a snob, I wasn't lying 😄

Comment (Kalma-House) [reply tests/es/climate_HassClimateGetTemperature.yaml:20]: Fixed. Thank you. Not a snob, just thorough, which I also try to be 😄 .

**Comment by davefx** [comment]
Please solve conflicts in order to review and possibly merge this PR

**Comment by davefx** [comment]
Extra sentences to get temperatures already added in PR #573 . Closing this PR.

**Pull request closed without merging**